### ISSUE TITLE
Use of YAML links

### ISSUE THREAD

**Issue by kmaustral** [SkepticMystic/graph-analysis#71]
I'm glad to discover this plug-in. 

Most of my links are in the YAML section. But these don't seem to be used in the algorithm. Is that correct? If so, is there a way to have them included?

**Comment by zergote** [comment]
Yes it works with the links in YALM but the full text does not appear in the panel.

Would it be feasible to have a switcher to activate the indexing in YALM code?

**Comment by lamborghinipth** [comment]
My question is whether the author keep developing the plugin or not? I do not see any updates. The latest is in 3 years ago.

**Comment by SkepticMystic** [comment]
I'm not actively developing this plugin. I would like to come back at some point, but absolutely no guarantees
